python/pacemaker/_cts/remote.py

```diff
@@ -116,29 +116,37 @@ def call_async(self, node, command, delegate=None):
         aproc.start()
         return aproc
 
-    def __call__(self, node, command, stdout=0, synchronous=1, silent=False, blocking=True, delegate=None):
-        '''Run the given command on the given remote system
-        If you call this class like a function, this is the function that gets
-        called.  It just runs it roughly as though it were a system() call
-        on the remote machine.  The first argument is name of the machine to
-        run it on.
-        '''
+    def __call__(self, node, command, synchronous=True, quiet=False, silent=False):
+        """ Run the given command on the given remote system.  If you call this class
+            like a function, this is what gets called.  It's approximately the same
+            as a system() call on the remote machine.
+
+            Arguments:
+
+            node        -- The remote machine to run on
+            command     -- The command to run, as a string
+            synchronous -- Should we wait for the command to complete?
+            quiet       -- If True, log the command and its return code but not
+                           its output
+            silent      -- If True, do not log anything - this implies quiet=True
+
+            Returns:
+
+            A tuple of (return code, command output)
+        """
```

Comment:
Thanks, the return value here has always been confusing.

What do you think about replacing quiet and silent with 0/1/2 verbose or log_level?